bin/gem

```diff
@@ -11,8 +11,4 @@ require 'rubygems/exceptions'
 
 args = ARGV.clone
 
-begin
-  Gem::GemRunner.new.run args
-rescue Gem::SystemExitException => e
```

Comment:
Since we're not directly using this exception constant, I think the require above for `rubygems/exceptions` can also be removed.